gellinas/adventures-blog
# Feature/admin dashboard

## Description
This adds the updates to handle some search features for the navbar and search page along with integrating updates from backend to support admin access control

## Test Steps
1. Confirm that admin access is working as expected
2. Confirm that searching via navbar and search page results in the correct responses
3. Confirm that admin dashboard is adding/updating adventures/photos
